src/System.IO.FileSystem.Watcher.Polling/System/IO/FileState.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics;
+
+namespace System.IO
+{
+    [Serializable]
+    struct FileState
```

Comment:
Accessibility modifier

Reply:
Is there a reason we don't have a solution-wide analyzer to remind us about style guidelines?

Reply:
> Is there a reason we don't have a solution-wide analyzer to remind us about style guidelines?

You do get the feedback in VS, but it isn't super in your face. I think that was intentional.

![image](https://user-images.githubusercontent.com/8184940/40090349-1c4b3268-5866-11e8-81c0-f6c2b30fbbfb.png)

In any case I'll bring it up with the team.